krkn/scenario_plugins/container/container_scenario_plugin.py

```diff
@@ -22,9 +22,7 @@ def run(
         lib_telemetry: KrknTelemetryOpenshift,
         scenario_telemetry: ScenarioTelemetry,
     ) -> int:
-        start_time = int(time.time())
```

Comment:
@paigerube14 is the start time and end time captured somewhere else for this scenario? Thoughts?

Reply:
Yes, it's in the main run kraken I believe. This section is duplicated between here and the scenario plugin abstract that it was waiting double the amount of time